src/js/components/Carousel/stories/Autoplay.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Attraction, Car, TreeOption } from 'grommet-icons';
+
+import { Text, Box } from 'grommet';
+import { Carousel } from '../Carousel';
+
+export const Autoplay = () => (
```

Comment:
I get a flash every so often at the end of a transition where it renders the child off to the left outside the container then immediately re-renders it correctly. Browser is chrome on windows.